index.html

```diff
@@ -293,16 +295,16 @@ <h2>Discussion servers</h2>
 
 <h3>Microsoft Teams</h3> 
 <p>
-<a href="https://teams.microsoft.com/l/team/19%3ae1a8302e08d24fd8b42154f6b8edbdbf%40thread.tacv2/conversations?groupId=2a4f0e48-af38-412e-90c5-a2c0a1284404&tenantId=e66e77b4-5724-44d7-8721-06df160450ce">The AutoRob Microsoft team</a> will be used for course-related discussions and announcements.  <a href="https://en.wikipedia.org/wiki/Microsoft_Teams">Microsoft Teams</a> is a cloud-hosted online discussion and collaboration system with functionality that resembles <a href="https://en.wikipedia.org/wiki/Internet_Relay_Chat">Internet Relay Chat (IRC)</a>.  Microsoft Teams clients are available for most modern operating systems as well as through the web.
+<a href="https://teams.microsoft.com/l/team/19%3ae1a8302e08d24fd8b42154f6b8edbdbf%40thread.tacv2/conversations?groupId=2a4f0e48-af38-412e-90c5-a2c0a1284404&tenantId=e66e77b4-5724-44d7-8721-06df160450ce">The AutoRob Microsoft team</a> will be used for course-related discussions and announcements.  <a href="https://en.wikipedia.org/wiki/Microsoft_Teams">Microsoft Teams</a> is a cloud-hosted online discussion and collaboration system with functionality that resembles <a href="https://en.wikipedia.org/wiki/Internet_Relay_Chat">Internet Relay Chat (IRC)</a>.  Microsoft Teams clients are available for most modern operating systems as well as through the web. Microsoft Teams is <a href="https://ro.umich.edu/records-registration/student-rights-records">FERPA</a> compliant.
```

Comment:
I think I remember you saying that MS Teams is FERPA compliant, unlike discord. That may be helpful for students to know.